### ISSUE TITLE
Could not locate index pattern

### ISSUE THREAD

**Issue by shikataganai89** [rocknsm/rock#139]
Hello
After a fresh installation, I opened HTTP dashboard and I got the following error:
"Could not locate that index-pattern-field (id:http.user_agent.keyword).
In the Saved Object->Edit Visualization I got the following error:
"There is a problem with this saved object. A field associated with this object no longer exists in the index pattern. "
I tried to change in the visState the field "field" in "http.user_agent" but I got:
"Visualize: Fielddata is disabled on text fields by default. Set fielddata=true on [http.user_agent] in order to load fielddata in memory by uninverting the inverted index. Note that this can however use significant memory".

How could I fix it?

**Comment by spartan782** [comment]
I'm not sure if this helps or not, but in the past when I have used this I have noticed that sometimes the field hasn't been generated yet due to none of that type of traffic existing in the index. The dashboards in kibana assume that all stated fields exist whether they do or don't. 

http.user_agent.keyword field is the correct field you will want to use. Have you tried pushing any traffic through it to generate some bro http logs before changing the field in kibana?
